src/views/home/index.js

```diff
@@ -1,16 +1,20 @@
 import { NavBar } from '../../components'
-import React, { useEffect, useState } from 'react'
+import React, { useEffect, useState, useContext } from 'react'
+import { AuthContext } from '../../providers/use-auth'
+import { useNavigate } from 'react-router'
 import './home.css'
 
 const getDog = async () => {
```

Comment:
I'm not sure about this part, looks like `getDog()` and other dog related stuff shouldn't be in `index.js`.  

But I'll give you a pass on this...!  
Maybe create an issue to refactor this...?

Reply:
I got it, thank you